command.go

```diff
@@ -404,3 +408,24 @@ func hasCommand(commands []*Command, command *Command) bool {
 
 	return false
 }
+
+func checkDuplicatedCmds(parent *Command) error {
+	if parent == nil {
+		return nil
+	}
+	if len(parent.Subcommands) == 0 {
+		return nil
+	}
```

Comment:
No need for this check either. `range` will skip empty list.

Reply:
sure, will delete it soon

Reply:
@abitrolly Please check again